src/app/project/ontology/default-data/default-properties.ts

```diff
@@ -131,22 +146,25 @@ export class DefaultProperties {
                 {
                     icon: 'link',
                     label: 'Other resource e.g. Person',
-                    subPropOf: 'knora-api:LinkValue',
-                    gui_ele: 'salsah-gui:Searchbox',    // 'Autocomplete',
-                    group: 'Link'
-                },
-                {
-                    icon: 'compare_arrows',
-                    label: 'External resource',
-                    subPropOf: 'knora-api:ExternalResValue',
-                    gui_ele: 'salsah-gui:SimpleText',
+                    subPropOf: Constants.HasLinkTo,
+                    objectType: Constants.Resource,
+                    gui_ele: Constants.SalsahGui + Constants.Delimiter + 'Searchbox',    // 'Autocomplete',
                     group: 'Link'
                 },
+                // {
```

Comment:
is this type waiting for something to make it operational?

Reply:
Yes this is something we have to discuss. I'm not sure if we need it. The same for duration and period. It does not exist in knora-api resp. in salsah gui ontology. It could be handled as gui attribute. I want to include client services into the discussion as soon, they can test it.
I just created DSP-1154 about this topic.